docs/osm-with-azure-policy.md

```diff
@@ -0,0 +1,21 @@
+## OSM with Azure Policy
+
+Adding OSM to an arc-enabled AKS cluster that has the Azure Policy addon installed can present some complications. The built-in Azure policy initiatives for kubernetes encompass several policies, some of which conflict with the OSM extension. For instance, without making any chances to the osm-controller or osm-label, the following policy violations blocked the OSM extension from being added. 
+
+
+- AllowedUsersGroups: 
```

Comment:
I think you can remove the rest of this section since we have patched the osm-controller deployment following security best practice and the arc-osm-system namespace is now excluded.